elasticsearch/client/ingest.py

```diff
@@ -46,16 +46,16 @@ def delete_pipeline(self, id, params=None):
             'pipeline', id), params=params)
```

Comment:
I guess the same should change should be applied in this method as well?

Reply:
Oh! Yes. I will change this ASAP. Thanks for your feedback!